parmed/openmm/parameters.py

```diff
@@ -116,7 +117,7 @@ def from_parameterset(cls, params, copy=False):
 
         return new_params
 
-    def write(self, dest, provenance=None):
+    def write(self, dest, provenance=None, write_unused=True):
```

Comment:
how is about changing `write_unused` to `include_unused` or something else.

it seems to be duplicated `def write(..., write_..)`